### ISSUE TITLE
Cloudflare 

### ISSUE THREAD

**Issue by SSett1** [KotatsuApp/kotatsu-parsers#637]
### Source information

Hipercool

### Steps to reproduce

1 - Click in Hipercool Scan
2 - Open the Cloudflare Screen
3 - Click in "Verification [...]"
4 - And de loading this page every time

### Kotatsu version

6.7.5

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Koitharu** [comment]
Update app to 6.8

**Comment by Priince26** [comment]
Updated to 7.3 and now getting this 
org.koitharu.kotatsu.core.exceptions.CloudFlareBlockedException: Blocked by CloudFlare
	at org.koitharu.kotatsu.core.network.GZipInterceptor.intercept(SourceFile:200)
	at okhttp3.internal.http.RealInterceptorChain.proceed(SourceFile:131)
	at org.koitharu.kotatsu.core.network.GZipInterceptor.intercept(SourceFile:251)
	at okhttp3.internal.http.RealInterceptorChain.proceed(SourceFile:131)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(SourceFile:97)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(SourceFile:48)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Blocked by cloudfare
![Screenshot_20240721_222533](https://github.com/user-attachments/assets/d076d63e-5228-4a71-992a-648326cde6be)
![Screenshot_20240721_222536](https://github.com/user-attachments/assets/05055b3d-836b-4241-8a2b-8db4fd1b873c)
